aws/_modules/iam/main.tf

```diff
@@ -32,7 +32,7 @@ resource "aws_iam_role" "TerraFailIAM_role" {
       {
         Effect = "Allow"
         Principal = {
-          AWS = "*"
+          AWS = "terrafail@user.com"
```

Comment:
Minor, but this is inconsistent with the previously defined fake user in aws/_modules/api_gateway/main.tf